image_generation/lcm_dreamshaper_v7/cpp/README.md

```diff
@@ -2,36 +2,54 @@
 
 The pure C++ text-to-image pipeline, driven by the OpenVINO native API for SD v1.5 Latent Consistency Model with LCM Scheduler. It includes advanced features like [LoRA](https://huggingface.co/docs/peft/main/en/conceptual_guides/lora#lora) integration with [safetensors](https://huggingface.co/docs/safetensors/index#format) and [OpenVINO Tokenizers](https://github.com/openvinotoolkit/openvino_tokenizers). Loading `openvino_tokenizers` to `ov::Core` enables tokenization. [The common folder](../../common/) contains schedulers for image generation and `imwrite()` for saving `bmp` images. This demo has been tested for Linux platform only. There is also a Jupyter [notebook](https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/notebooks/latent-consistency-models-image-generation/lcm-lora-controlnet.ipynb) which provides an example of image generaztion in Python.
 
-> [!NOTE]
-> This tutorial assumes that the current working directory is `<openvino.genai repo>/image_generation/lcm_dreamshaper_v7/cpp/` and all paths are relative to this folder.
+## Software Requirements
+
+### Linux
 
-## Step 1: Prepare Build Environment
+- [CMake](https://cmake.org/download/) 3.23 or higher
+- GCC 7.5 or higher
+- Python 3.8 or higher
+- Git
 
-Prerequisites:
-- Conda ([installation guide](https://conda.io/projects/conda/en/latest/user-guide/install/index.html))
+### Windows
 
-C++ Packages:
-* [CMake](https://cmake.org/download/): Cross-platform build tool
-* [OpenVINO](https://docs.openvino.ai/2024/get-started/install-openvino.html): Model inference
+- [CMake](https://cmake.org/download/) 3.23 or higher
+- Microsoft Visual Studio 2019 or higher, version 16.3 or later
+- Python 3.8 or higher
+- Git for Windows
```

Comment:
```suggestion
- Git for Windows
- [NSIS](https://sourceforge.net/projects/nsis/)
```
I'm not sure if it's needed. If you are unsure as well, adding it would be safer.

Reply:
agree. I saw the same in other readmes, please, clean up everywhere